src/NHibernate/Impl/AbstractSessionImpl.cs

```diff
@@ -444,5 +446,26 @@ internal IOuterJoinLoadable GetOuterJoinLoadable(string entityName)
 		public abstract IEnumerable<T> Enumerable<T>(IQueryExpression queryExpression, QueryParameters queryParameters);
 
 		public abstract int ExecuteUpdate(IQueryExpression queryExpression, QueryParameters queryParameters);
+		
+		/// <summary>
+		/// Creates a new Linq <see cref="IQueryable{T}"/> for the entity class.
+		/// </summary>
+		/// <typeparam name="T">The entity class</typeparam>
+		/// <returns>An <see cref="IQueryable{T}"/> instance</returns>
+		public IQueryable<T> Query<T>()
```

Comment:
I now tend to just put `/// <inheritdoc />` for concrete implementation, in order to avoid duplicating doc, although it is not supported by all tools (actually for what NHibernate use, only sandcastle supports it I believe). What do you think of that?

Reply:
It doesn't work this way in this case as AbstractSessionImpl does not implement either of these interfaces.